src/execution/operator/aggregate/physical_simple_aggregate.cpp

```diff
@@ -207,5 +210,7 @@ string PhysicalSimpleAggregate::ParamsToString() const {
 	}
 	return result;
 }
+void PhysicalSimpleAggregate::FinalizeOperatorState(PhysicalOperatorState &state, ExecutionContext &context) {
```

Comment:
Why does `FinalizeOperatorState` need to be overridden here? Do you need the function of the base class not to be called?

Reply:
For this PR, FinalizeOperatorState does not need to be overridden. But there is an aggregate function in this operator that I think needs to be monitored. Therefore FinalizeOperatorState needs to be overridden to flush down the monitored information.